src/components/Contacts/ContactDetails/ContactDontationsTab/PartnershipInfo/EditPartnershipInfoModal/EditPartnershipInfoModal.tsx

```diff
@@ -242,38 +374,41 @@ export const EditPartnershipInfoModal: React.FC<EditPartnershipInfoModalProps> =
                   <InputLabel id="currency-select-label">
                     {t('Currency')}
                   </InputLabel>
-                  <Select
-                    labelId="currency-select-label"
-                    value={pledgeCurrency}
-                    onChange={handleChange('pledgeCurrency')}
-                    MenuProps={{
-                      anchorOrigin: {
-                        vertical: 'bottom',
-                        horizontal: 'left',
-                      },
-                      transformOrigin: {
-                        vertical: 'top',
-                        horizontal: 'left',
-                      },
-                      getContentAnchorEl: null,
-                      PaperProps: {
-                        style: {
-                          maxHeight: '300px',
-                          overflow: 'auto',
+                  {pledgeCurrencies && (
+                    <Select
+                      labelId="currency-select-label"
+                      value={pledgeCurrency ?? ''}
+                      onChange={handleChange('pledgeCurrency')}
+                      MenuProps={{
+                        anchorOrigin: {
+                          vertical: 'bottom',
+                          horizontal: 'left',
                         },
-                      },
-                    }}
-                  >
-                    {pledgeCurrencies?.map(
-                      ({ value, id }) =>
-                        value &&
-                        id && (
-                          <MenuItem key={id} value={id}>
-                            {t(value)}
-                          </MenuItem>
-                        ),
-                    )}
-                  </Select>
+                        transformOrigin: {
+                          vertical: 'top',
+                          horizontal: 'left',
+                        },
+                        getContentAnchorEl: null,
+                        PaperProps: {
+                          style: {
+                            maxHeight: '300px',
+                            overflow: 'auto',
+                          },
+                        },
+                      }}
+                    >
+                      <MenuItem value={''} disabled></MenuItem>
```

Comment:
I was getting errors about the value not existing in the select. Turns out MUI Select components do not support ```null``` and they don't plan on it. So you have to turn the value into an empty string and then create this dummy menu item so the error won't happen.